cylc/flow/cfgspec/globalcfg.py

```diff
@@ -551,10 +553,10 @@
         with Conf('<group>'):
             Conf('platforms', VDR.V_STRING_LIST)
     # Symlink Dirs
-    with Conf('symlink dirs', desc="""Define directories to be moved, symlinks
-                from the the original ``$HOME/cylc-run`` directories will be
-                created.
-            """):
+    with Conf('symlink dirs', desc="""
+        Define directories to be moved, symlinks from the the original
+        ``$HOME/cylc-run`` directories will be created.
+    """):
```

Comment:
Always start the string on the next line otherwise each line will get prefixed with whitespace (it gets passed through dedent).